application/src/main/resources/db/V13__Add_Reminder_Failure_attempts.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pending_reminders ADD failure_attempts INTEGER DEFAULT 0 NOT NULL ;
```

Comment:
i believe not null should be before default

Reply:
checking other examples in our code base, it doesnt seem to matter. the only odd thing is the space before the semicolon `;`